pkg/build/nodeimage/const_storage.go

```diff
@@ -23,10 +23,11 @@ NOTE: we have customized it in the following ways:
 - our own image and helper image
 - schedule to linux nodes only
 - install as the default storage class
+- tolerate control plane scheduling taints
 */
 
-const storageProvisionerImage = "docker.io/kindest/local-path-provisioner:v20241108-5c6d2daf"
-const storageHelperImage = "docker.io/kindest/local-path-helper:v20230510-486859a6"
+const storageProvisionerImage = "docker.io/kindest/local-path-provisioner:v20241212-8ac705d0"
+const storageHelperImage = "docker.io/kindest/local-path-helper:v20241212-8ac705d0"
 
 // image we need to preload
 var defaultStorageImages = []string{storageProvisionerImage, storageHelperImage}
```

Comment:
they also reworked the rbac upstream, I don't think it's strictly necessary but I meant to port that change over too (I'm re-applying the diff from v0.0.24 .. v0.0.30 while maintaining our customizations)